CONTRIBUTING.md

```diff
@@ -82,6 +82,7 @@ The first time you run the app, simply run:
 npm install # installs node modules
 docker-compose up # Spin up docker containers
 npm run db # Creates DBs and initializes tables
+npm run sqs:init #initialises localstack sqs queues
```

Comment:
We need to initialise localstack queue before using the local app. I didn't test it with ngrok (only with atlastunnel)